test-data/unit/lib-stub/dataclasses.pyi

```diff
@@ -5,26 +5,29 @@ _T = TypeVar('_T')
 class InitVar(Generic[_T]):
     ...
 
+class _KW_ONLY_TYPE: ...
+KW_ONLY = _KW_ONLY_TYPE
```

Comment:
Not entirely sure, but https://github.com/python/typeshed/pull/5826 changed how `KW_ONLY` is defined.
After the typeshed changes have been synced, I think this needs to be updated.